src/annotator/components/ContentInfoBanner.js

```diff
@@ -52,9 +58,9 @@ export default function ContentInfoBanner({ info }) {
           'font-semibold'
         )}
         data-testid="content-container-info"
-        title={info.item.containerTitle}
+        title={info.container.title}
```

Comment:
After these changes land, we can stop sending over `contentTitle` at the top level of the object from LMS.